libs/designsystem/src/lib/components/fab-sheet/fab-sheet.component.scss

```
@@ -9,6 +10,14 @@ $fab-sheet-margin: utils.size('s');
 // Ionic fab can be customized by using the css properties listed in the CSS Custom Properties section of the below docs page:
 // https://ionicframework.com/docs/api/fab-button
 ion-fab-button {
+  @include ionic.apply-hover;
+
+  // NOTE: Also see ion-back-button in page component and ion-segment-button in segmented control component
+  &::part(native)::after {
+    @include ionic.transition;
+  }
+  // end NOTE
```

What should i note when looking at ion-back-button in page component and ion-segment-button in segmented control component? Consider elaborating.

---

If we could add hyperlinks to code it would be great. The note is just an attempt to refer to the other places we use the same custom implementation of hover.

I hope we can consolidate this somehow when we finish the remaining issues for hover, where we need special solutions for different ways to make it work for Ionic components.

---

Aaaah i see! I took a stab at reformulating it :) 
```suggestion
  /* 
  NOTE: this is a custom implementation of the hover interaction state; 
  identical to PageComponent's ion-back-button & SegmentedControlComponent's 
  ion-segment-button. 
  */
  &::part(native)::after {
    @include ionic.transition;
  }
  // end NOTE
```